pyop2/types/dat.py

```diff
@@ -492,6 +492,20 @@ def norm(self):
         from math import sqrt
         return sqrt(self.inner(self).real)
 
+    def axpy(self, alpha: float, other: 'Dat') -> None:
+        """Compute the operation :math:`y = \\alpha x + y`.
+
+        :arg alpha: a scalar
+        :arg other: the :class:`Dat` to add to this one
```

Comment:
Usually this argument would be called `x` to be clear what role it has in the y=a*x+y procedure. If `other` is the standard argument name for `Dat` methods then the fact that `other` is x and `self` is y should be stated in the docstring.